docs/static/troubleshooting.asciidoc

```diff
@@ -48,13 +48,29 @@ Operation not permitted
 [[ts-startup]] 
 == {ls} start up
 
+[float] 
+[[ts-java-home]] 
+=== [add decription] errors
```

Comment:
@robbavey What kind of errors would one see if JAVA_HOME is not set?

Reply:
It depends...

When running Logstash, there is an attempt to try and find java on the path if `JAVA_HOME` is not found. This may be successful, and would see Logstash running with the version of Java returned by `command -v java` on Linux/MacOs or the first `java.exe` on the path in Windows.

If no java is found on the path, then the user may see this message:
```could not find java; set JAVA_HOME or ensure java is in PATH```

This will typically happen when Java is not installed, or has been installed via a tar ball. I'm also trying to track down if there is an edge case where this can happen where java has been installed regularly, but `JAVA_HOME` has not been set.